moveit_ros/planning/kinematics_plugin_loader/include/moveit/kinematics_plugin_loader/kinematics_plugin_loader.h

```diff
@@ -102,17 +80,14 @@ class KinematicsPluginLoader
 
 private:
   const rclcpp::Node::SharedPtr node_;
+  std::unordered_map<std::string, std::shared_ptr<kinematics::ParamListener>> kinematics_param_listener_;
+  std::unordered_map<std::string, kinematics::Params> kinematics_params_;
```

Comment:
These are already inside the kinematics_plugin_loader, so I don't think you need to restate the name kinematics in the name for these. More importantly, I think the name should include what the key of the map is.

```suggestion
  std::unordered_map<std::string, std::shared_ptr<kinematics::ParamListener>> group_param_listener_;
  std::unordered_map<std::string, kinematics::Params> group_params_;
```